### PR TITLE
chore: update react-query example

### PR DESCRIPTION
**Pull request by Pagebakers**
Updated the example to @wundergraph/react-query@0.2.0
Removed secrets configuration.

### CODE REVIEW

File: examples/nextjs-react-query/.wundergraph/wundergraph.config.ts [hunk 1]
```diff
@@ -37,9 +29,6 @@ configureWunderGraphApplication({
 		cookieBased: {
 			providers: [authProviders.demo()],
 			authorizedRedirectUris: ['http://localhost:3000'],
-			secureCookieHashKey: new EnvironmentVariable('WUNDERGRAPH_SECURE_COOKIE_HASH_KEY'), // must be of length 32
```
Comment (JivusAyrus) [inline examples/nextjs-react-query/.wundergraph/wundergraph.config.ts:40]: We need this for the tests

**Review by JivusAyrus — CHANGES_REQUESTED** [review]
Can we change the name in the package.json?  I forgot to do it last time

File: examples/nextjs-react-query/.wundergraph/wundergraph.config.ts (hunk 1 above)
Comment (Pagebakers) [reply examples/nextjs-react-query/.wundergraph/wundergraph.config.ts:40]: This is included in all examples now?

Comment (JivusAyrus) [reply examples/nextjs-react-query/.wundergraph/wundergraph.config.ts:40]: All examples which use authentication

**Review by JivusAyrus — APPROVED** [review]

File: examples/nextjs-react-query/.wundergraph/wundergraph.config.ts (hunk 1 above)
Comment (Pagebakers) [reply examples/nextjs-react-query/.wundergraph/wundergraph.config.ts]: Ok I thought it was some leftover code from copying another example. This one doesn't use auth. But I'll leave it in for now then.

**Review by fiam — APPROVED** [review]

**Pull request merged**